acts

```diff
@@ -180,37 +179,41 @@ if [ "$backuprc" != "0" ]; then
     die "acts-tarsnap-error One of the backups failed -- not deleting old backups"
 fi
 
-# We don't delete any yearly backups
-
-# We keep 12 monthly backups
-monthlybackups=$(echo "$archives" | grep "$hostname-monthly-" | cut -d_ -f1 | uniq | sort -rn)
-if [ "$(echo "$monthlybackups" | wc -l)" -gt 12 ]; then
-    log_debug 'message="More than 12 monthly backups, deleting the oldest"'
-    echo "$monthlybackups" | tail -n +13 | while read -r archiveprefixtodel; do
-        log_verbose "message=\"Deleting backup prefix $archiveprefixtodel*\""
-        echo "$archives" | grep -E "^$archiveprefixtodel" | while read -r archivetodel; do
-            log_debug "message=\"Deleting backup $archivetodel\""
-            $tarsnap -d -f "$archivetodel"
+
+for dir in $backuptargets; do
```

Comment:
Can you add a log_debug message specifying which backup target is being checked at the start of the loop?